WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/creation/UnifiedOrderEditViewModelTest.kt

```diff
@@ -172,6 +176,99 @@ abstract class UnifiedOrderEditViewModelTest : BaseUnitTest() {
         )
     }
 
+    @Test
+    open fun `when product quantity increased, send tracks event`() {
+        val productId = 1L
+        val products = OrderTestUtils.generateTestOrderItems(count = 1, productId = productId)
+        val order = defaultOrderValue.copy(items = products)
+        initMocksForAnalyticsWithOrder(order)
+        createSut()
+        sut.onIncreaseProductsQuantity(productId)
+        verify(tracker).track(
```

Comment:
Minor thing 😅  but I think that generally we followed given/when/then convention in the codebase, so I'd suggest adding at least empty spaces between those sections for easier reading.

```suggestion
        initMocksForAnalyticsWithOrder(order)
        createSut()
        
        sut.onIncreaseProductsQuantity(productId)
        
        verify(tracker).track(
```